compiler/AST/expr.cpp

```diff
@@ -97,7 +97,7 @@ static void codegenCall(const char* fnName, GenRet a1, GenRet a2, GenRet a3, Gen
 static void codegenCall(const char* fnName, GenRet a1, GenRet a2, GenRet a3, GenRet a4, GenRet a5);
 
 static GenRet codegenZero();
-static GenRet codegenOne();
+//static GenRet codegenOne();
```

Comment:
Yeah, sometimes those "unused" messages are useful and sometimes they're a downright nuisance. Holding on to it briefly seems wise to me.